README.rst

```diff
@@ -12,6 +12,11 @@ Collaborative Brain Wave Analysis Pipeline (Cobrawap)
   :alt: DOI Latest Release
   :align: left
 
+.. image:: https://img.shields.io/badge/slack-join-pink.svg
+   :target: https://join.slack.com/t/cobrawapworkinggroup/shared_invite/zt-1t8fjv447-64MrlHywww97LRC1ZtW0DA
+   :alt: Join our Slack Community
+   :align: left
```

Comment:
```suggestion
   :align: right
```

Reply:
not sure if "align left" would work when displayed on screen, because there are three banners in the same line now

Reply:
The alignment to the left for all the elements has the effect that they are listed one after the other from left to right. You can see the rendered version of the README on the fork branch.
However, it seems that the displayed order (doi, slack, docs) is not the same as in the rst code (docs, doi, slack). I'm not sure how the display order is determined then.

Reply:
I can't see your fork branch (I see there are 14 forks, but just mine is visible to me); my preview of the .rst code here in the github review tool shows the "slack" banner in a new line, below the others